main.go

```diff
@@ -56,7 +56,7 @@ func run() error {
 		return fmt.Errorf("failed to render template: %v", err)
 	}
 
-	fmt.Printf("::set-output name=result::%s", escape(output))
+	os.Setenv("GITHUB_OUTPUT", fmt.Sprintf("result=%s", escape(output)))
```

Comment:
Hmm, does this overwrite GITHUB_OUTPUT? Don't we want to just append to it?

Reply:
> Hmm, does this overwrite GITHUB_OUTPUT? Don't we want to just append to it?

Yes, this overwrites the `GITHUB_OUTPUT`. Since we set it only in one place, I decided to overwrite it. 

I'll look into how to append to it so we don't accidentally overwrite other existing env vars.

Reply:
Okay, looks like this works. I tested it here https://go.dev/play/p/urtPbvAYgH1:

```suggestion
	os.Setenv("GITHUB_OUTPUT", fmt.Sprintln(os.Getenv("GITHUB_OUTPUT"))+fmt.Sprintf("result=%s", escape(output)))
```